### DISCUSSION TITLE
document lua version and changes

### DISCUSSION THREAD

**Discussion by atesin** [nesbox/TIC-80#2194] [Ideas]
hi... i dont know what lua version tic80 runs (or other supported languages), or what changes/customizations has been made (like integer division "//") or what lua api/lib had been implemented (all?)

i think is relevant when we are making some game/app and need to read lua (or other) documentation, to know how to do something... for example lua has many versions and all of them have differences between (different commands, operators, etc)

**Comment by nesbox** [comment]
We are using `v5.3.6` but yes, it could be documented somewhere.

**Comment by atesin** [comment]
for example....  what is the behavior with integer division "//" with negative decimals?? ....  i think it should advance to next higher integer, to keep function continuity... but if just truncate decimals...  i dont know

UPDATE: i tried myself with `eval trace(-100/9)` and `eval trace(-100//9)` and confirmed it keeps continuity (approximates to LOWER integer, regardless the sign)

IT WORTHS TO NOTE i had this problem while running eval = https://github.com/nesbox/TIC-80/issues/800  ....  i like tic80, but console command documentation is also awful
